tests/components/automation/test_event.py

```diff
@@ -143,7 +143,7 @@ async def test_if_fires_on_event_with_nested_data(hass, calls):
         "test_event", {"parent_attr": {"some_attr": "some_value", "another": "value"}}
     )
     await hass.async_block_till_done()
-    assert 1 == len(calls)
+    assert len(calls) == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5456697)